lib/apis/moray.js

```diff
@@ -586,8 +596,25 @@ Moray.prototype.putVm = function (uuid, vm, cb) {
     this.connection.putObject(VMS_BUCKET_NAME, uuid, object, cb);
 };
 
+/*
+ * Deletes *all* VMs that match the filter generated from "params".
+ */
```

Comment:
It may be worth noting in the comment for `".delVms()"` that we
don't seem to actually allow record deletion for VMs -- we just
mark them destroyed.  It looks like this method is really just
for the test harness; is that correct?